python/GafferSceneUI/_InspectorColumn.py

```diff
@@ -176,43 +176,18 @@ def __disablableInspectionTweaks( pathListing ) :
 
 			with inspectionContext :
 				inspection = column.inspector().inspect()
-				if inspection is not None and inspection.editable() :
```

Comment:
I'm not sure how this is happening, but the right-click menu for a read-only edit is emitting this error in the shell and showing an incomplete menu rather than showing a greyed-out "Disabled Edit" item :

```ERROR : Emitting signal : Not editable : RectLight.visualiserAttributes.scale is locked.```

That's editing the scale on the light below :

<details>

```
import Gaffer
import GafferScene
import GafferUSD
import imath

Gaffer.Metadata.registerValue( parent, "serialiser:milestoneVersion", 1, persistent=False )
Gaffer.Metadata.registerValue( parent, "serialiser:majorVersion", 5, persistent=False )
Gaffer.Metadata.registerValue( parent, "serialiser:minorVersion", 0, persistent=False )
Gaffer.Metadata.registerValue( parent, "serialiser:patchVersion", 0, persistent=False )

__children = {}

__children["RectLight"] = GafferUSD.USDLight( "RectLight" )
parent.addChild( __children["RectLight"] )
__children["RectLight"].loadShader( "RectLight" )
__children["RectLight"].addChild( Gaffer.V2fPlug( "__uiPosition", defaultValue = imath.V2f( 0, 0 ), flags = Gaffer.Plug.Flags.Default | Gaffer.Plug.Flags.Dynamic, ) )
__children["RectLight"]["name"].setValue( 'rectLight' )
__children["RectLight"]["parameters"]["arnold:aov"]["enabled"].setValue( True )
__children["RectLight"]["parameters"]["shaping:cone:angle"]["value"].setValue( 25.0 )
__children["RectLight"]["visualiserAttributes"]["scale"]["enabled"].setValue( True )
Gaffer.Metadata.registerValue( __children["RectLight"]["visualiserAttributes"]["scale"], 'readOnly', True )
__children["RectLight"]["__uiPosition"].setValue( imath.V2f( -5.75, 9.85000038 ) )


del __children

```

</details>

Reply:
Likewise, the `D` shortcut is emitting this message to the shell rather than showing a pop-up message :

```ERROR : EventSignalCombiner : Not editable : RectLight.visualiserAttributes.scale is locked.```

Reply:
Thanks for finding that one, I'm now properly elevating that `RectLight.visualiserAttributes.scale is locked.` message to the user in the form of a pop-up as part of the overall messaging improvements in 1968cbf.